server/src/test/java/org/opensearch/action/admin/indices/mapping/get/GetFieldMappingsResponseTests.java

```diff
@@ -72,6 +74,51 @@ public void testNullFieldMappingToXContent() {
         assertEquals("{\"index\":{\"mappings\":{}}}", Strings.toString(MediaTypeRegistry.JSON, response));
     }
 
+    public void testBwcSerializationWithNullField() throws Exception {
```

Comment:
We probably don't need to have such serde targeted tests, `MixedClusterClientYamlTestSuiteIT` covers that, right?

Reply:
Actually yes, the yaml test can cover that, I've removed this test.